bin/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
This has some common bootstrapping (lines 1-8, 28-34) but most of this file deals with handling the (mostly project-specific) process for building the project artifacts themselves. For example this Cassandra build starts off similar for lines 1-15, but the build itself is different and the resulting paths/artifacts passed to `ci_upload.py` are different:
https://github.com/mesosphere/dcos-cassandra-service/blob/master/build.sh